frontend/next.config.js

```diff
@@ -47,13 +47,42 @@ const moduleExports = {
       ];
    },
    async redirects() {
-      return [
+      const redirects =[
          {
             source: '/Store/Guide/:guideid',
             destination: `${process.env.NEXT_PUBLIC_IFIXIT_ORIGIN}/Guide/_/:guideid`,
             permanent: true,
          },
       ];
+
+      if (process.env.FORCE_SUBDOMAIN_REDIRECTS !== "true") {
+         return redirects;
+      }
```

Comment:
Not a big deal, but I like the flow to test the "positive" condition instead of having a negation (reason being negative conditions are generally harder to grasp)..
I find the code a little bit easier to follow this way, but feel free to ignore this suggestion

```suggestion
      if (process.env.FORCE_SUBDOMAIN_REDIRECTS === "true") {
         redirects.concat(...);
      }
      return redirects;
```